### ISSUE TITLE
taxonomy search not returning returning

### ISSUE THREAD

**Issue by brettin** [PATRIC3/patric3_website#856]
From search taxa i entered

Planctomycetes Verrucomicrobia Chlamydiae

The results never finished loading. Note the "loading" message in green:

![image](https://cloud.githubusercontent.com/assets/991769/17755680/f1a735da-64a0-11e6-80d5-c88cb82781ac.png)

Here is the url:

https://www.alpha.patricbrc.org/view/TaxonList/?and(keyword(Planctomycetes),keyword(Verrucomicrobia),keyword(Chlamydiae))

**Comment by dmachi** [comment]
We’ll need to make this handle ‘no results’ better.  That is the problem here.  If you do that same search for “everything”, you’ll see that there are no matching results.  So it is saying loading, but it really already tried and didn’t find anything.

I don’t really know anything about these taxa, but are you trying to target a single taxa or where you trying to get back three?

**Comment by mshukla1** [comment]
Agree with Dustin. 

However, another issue here is that Tom entered three different taxon names and was hoping to find all three in a single search operation, instead of trying them one by one.

We need to support batch query operation with multiple keywords or ids, or provide functionality to change default search logic from AND to OR to achieve similar effect. 

This issue is connected to more broader ticket #896

**Comment by dmachi** [comment]
Fixed this specific issue so that it properly shows "0 Taxa" in the view header and "No Results Found." in the table.  I also changed the default grid behavior (for all grids inheriting from PageGrid, which should be most of them) so that they show "No Results Found." when there is no data found.

The other problem of and/or for the search is being addressed in the broader ticket, but this can be reviewed and closed now.
